### ISSUE TITLE
unable to launch RPM after browser completes oauth flow

### ISSUE THREAD

**Issue by askgh** [shiftkey/desktop#535]
### Version & OS

Version 2.9.0-linux1 (x64)

use rpm -Uvh install 2.9.0.rpm

and use root to login graphic ui

and chrome luanch with --no-sandbox
and github-desktop luanch iwth --no-sandbox

### Steps to reproduce the behavior

1. Go to options
2. Click on github.com signin
3. use chrome to login in github.com
4. chrome can't back to github desktop
5. get File not found: x-github-desktop-dev-auth://oauth*******************************
<img width="426" alt="1" src="https://user-images.githubusercontent.com/72971953/122901968-a5733e80-d380-11eb-93e8-5428379170ca.png">

i try some step to solve this problem
use #xdg-settings set default-url-scheme-handler x-github-desktop-dev-auth github-desktop.desktop
to avoid chrome can't back to github
after this setting. chrome can back to github-desktop. but the ui of github-desktop is hang on login
the part log os github-desktop is here

2021-06-23T02:27:58.802Z - info: [main] opening in browser: https://github.com/login/oauth/authorize?****************************************
2021-06-23T02:28:02.196Z - info: [main] Received possible protocol arguments: 3
2021-06-23T02:28:02.196Z - info: [main] Processing protocol url
2021-06-23T02:28:02.205Z - warn: [ui] Unknown URL action: unknown - payload: {"name":"unknown","url":"--no-sandbox"}

<img width="677" alt="1" src="https://user-images.githubusercontent.com/72971953/123026301-802d1180-d40e-11eb-83ae-97ecdf5edc99.png">

**Comment by shiftkey** [comment]
@ask-github which installer are you using?

**Comment by askgh** [comment]
> @ask-github which installer are you using?

update the problem description

thanks very much

**Comment by shiftkey** [comment]
> Version 2.9.0-linux1 (x64)

```
2021-06-23T02:28:02.205Z - warn: [ui] Unknown URL action: unknown - payload: {"name":"unknown","url":"--no-sandbox"}
```

Please retest with a newer version as I believe this was fixed with [`2.9.9-linux3`](https://github.com/shiftkey/desktop/releases/tag/release-2.9.9-linux3).
